### PR TITLE
Fixing CI skipping for automated PRs

### PR DESCRIPTION
**Pull request by bsipocz**
Skipping ci comment has to be in commit message and not in the title of the PR

### CODE REVIEW

**Comment by bsipocz** [comment]
Oh, well. In the meantime I saw #166, so now am updating this PR to remove the mentioning of skipping testing instead. 

(The confusing bit was that I saw on travis that PRs are running with ``[ci skip]`` in their title, and their description also say that.)

File: scripts/regenerate_feedstock.py [hunk 1]
```diff
@@ -206,15 +206,15 @@ def create_update_pr(clone, remote_head, fork_remote, upstream_remote):
...
 
 Remember, for any changes to the recipe you would normally need to increment the version or the build number of the package.
-Since this is an infrastructural change, we don't actually need/want a new version to be uploaded to anaconda.org/conda-forge, so the version and build/number are left unchanged and the CI has been skipped.
+Since this is an infrastructural change, we don't actually need/want a new version to be uploaded to anaconda.org/conda-forge, so the version and build/number are left unchanged.
 
 Thanks!
 
                     """.format(conda_smithy.__version__))
-            pull = forge_feedstock.create_pull(title='MNT: Re-render the feedstock [ci skip]',
+            pull = forge_feedstock.create_pull(title='MNT: Re-render the feedstock',
```
Comment (jakirkham) [inline scripts/regenerate_feedstock.py]: Are we adding this back?

Comment (bsipocz) [inline scripts/regenerate_feedstock.py]: Current PRs have this in their title, which was basically the source of my confusion. Was it there for any reason I missed?

Comment (jakirkham) [inline scripts/regenerate_feedstock.py]: It skips the build on merge.

Comment (bsipocz) [inline scripts/regenerate_feedstock.py]: Oh. I didn't know. In that case, I'm sorry for the noise.

Comment (jakirkham) [inline scripts/regenerate_feedstock.py]: Yeah, sorry, we missed changing the description.

Comment (jakirkham) [inline scripts/regenerate_feedstock.py]: No worries. Definitely appreciate the effort of looking for more places to optimize.

**Pull request closed without merging**